apps/www/components/ActionBar/index.js

```diff
@@ -242,7 +245,7 @@ const ActionBar = ({
         />
       ),
       modes: ['articleOverlay', 'feed', 'bookmark', 'seriesEpisode'],
-      show: !!document,
+      show: !!document && document.userProgress,
```

Comment:
this was a bug that I only detected now. Actionbar would render an empty UserProgress component, even if userProgress was null

Reply:
That was because [UserProgress](https://github.com/republik/plattform/blob/f1d5bf64bd7de69b57b2f6dd800cc12a1604fac0/apps/www/components/ActionBar/UserProgress.js#L78-L82) will return `null` if `userProgress` is falsy and thus attempting to [render an `IconButton` with too few props](https://github.com/republik/plattform/blob/f1d5bf64bd7de69b57b2f6dd800cc12a1604fac0/apps/www/components/ActionBar/index.js#L492-L496)?